external/anomaly/ote_anomalib/exportable_code/anomaly_classification.py

```diff
@@ -1,54 +1,22 @@
-"""Wrapper for Open Model Zoo for Anomaly tasks."""
```

Comment:
Should this file be placed under `anomaly_classification` instead of `ote_anomalib`?

Reply:
Yeah. I think that would make more sense. I'll move this file.